src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import {useState} from 'react';
+
+export default function FormPage() {
```

Comment:
the function name is misleading, because you are coding a  component but name it as a page